### ISSUE TITLE
Unable to package Meka 1.9.2 or Snapshot

### ISSUE THREAD

**Issue by gb-git** [Waikato/meka#53]
I have tried to package Meka 1.9.2 with no success.

I am following the command in the tutorial:
 mvn clean install

And I get the following stack trace:
```
[INFO] --- latex-maven-plugin:1.4.1:latex (default) @ meka ---
[ERROR]
java.io.IOException: Cannot run program "pdflatex" (in directory "F:\Projects\meka\target\latex\Tutorial"): CreateProcess error=2, The system cannot find the file specified
        at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
        at java.lang.Runtime.exec(Runtime.java:620)
…
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
        at java.lang.ProcessImpl.create(Native Method)
        at java.lang.ProcessImpl.<init>(ProcessImpl.java:386)
        at java.lang.ProcessImpl.start(ProcessImpl.java:137)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 04:16 min
[INFO] Finished at: 2018-10-12T12:18:50-07:00
[INFO] Final Memory: 387M/1212M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.fracpete:latex-maven-plugin:1.4.1:latex (default) on project meka: Cannot run program "pdflatex" (in directory "F:\Projects\meka\target\latex\Tutorial"): CreateProcess error=2, The system cannot find the file specified -> [Help 1]
```

I expect the project to compile and provide me with an environment with the BAT file, lib directory with needed dependencies and successfully built meka-X.X.X.jar file.

I am using Windows 8.1. Pro on an Intel P7 computer with 16GB of RAM. I am compiling with Java Oracle JDK 1.8.0_181 under cygwin (I have built many other projects with success, so I do not think the problem is in my cygwin setup).

Note that I also tried with Meka snapshot (Latest commit fe5eeb5) with no success.

Any help would be greatly appreciated.

**Comment by fracpete** [comment]
You don't seem to have `pdflatex` on your system PATH. This executable is used to turn the LaTeX documentation into PDFs. On my Linux Mint 18.2, I use texlive (http://tug.org/texlive/), but you should be able to use MikTeX on Windows as well (https://miktex.org/). Once you have installed your LaTeX distribution, make sure that the `pdflatex` is accessible via the PATH environment variable.

**Comment by gb-git** [comment]
Thank you fracpete. I installed miktex-2.9.6850 and configured the path as suggested. When I tried to build again (mvn clean install), It processed several files with pdflatex.exe, but failed with the following error:
```
! LaTeX Error: File `epsf.sty' not found.
```

I found a different way to get it to work:

Need to add this plugin to get the JARs in local directory.
```
<plugin> 
<groupId>org.apache.maven.plugins</groupId> 
<artifactId>maven-dependency-plugin</artifactId> 
<configuration> 
<outputDirectory> ${project.build.directory} </outputDirectory> 
</configuration> 
</plugin>
```
Then run 
- mvn clean install -DskipTests
- mvn dependency:copy-dependencies
- Move ./target/dependency directory to ./lib
- Copy ./target/meka-1.9.2.jar to ./lib
- Added run.bat file
```
@ECHO OFF
SET MEMORY=512m
SET MAIN=meka.gui.guichooser.GUIChooser
java -Xmx%MEMORY% -cp ".\lib\*" %MAIN% %1
```

Then execute the run.bat file which launches Meka. I hope this workaround is useful for someone else.
